.env

```diff
@@ -61,10 +61,11 @@ ENABLE_ANONYMOUS_COMMENTS=false
 
 MAX_AUTO_RENEW_SUBSCRIPTION_BATCH=false
 
-BEXIO_API_KEY=secret
-BEXIO_USER_ID=1
+BEXIO_API_KEY=eyJraWQiOiI2ZGM2YmJlOC1iMjZjLTExZTgtOGUwZC0wMjQyYWMxMTAwMDIiLCJhbGciOiJSUzI1NiJ9.eyJzdWIiOiJtYXRoaWFzLnN0cmVpdEBoYXVwdHN0YWR0LmJlIiwibG9naW5faWQiOiI5ZmNlMTNkOS1hN2ZhLTRiZmItOWFhNy03ZmI0NjY1ZTMxOTMiLCJjb21wYW55X2lkIjoiam41ZXlmejF2dmV1IiwidXNlcl9pZCI6MzE0MDMzLCJhenAiOiJldmVybGFzdC10b2tlbi1vZmZpY2UtY2xpZW50Iiwic2NvcGUiOiJvcGVuaWQgcHJvZmlsZSBlbWFpbCBhbGwgdGVjaG5pY2FsIiwiaXNzIjoiaHR0cHM6XC9cL2lkcC5iZXhpby5jb20iLCJleHAiOjMyNzg2NjQ4NDksImlhdCI6MTcwMTg2NDg0OSwiY29tcGFueV91c2VyX2lkIjoyLCJqdGkiOiI0OGM5ODg1ZS1hOTI2LTRlZTQtOTU0Zi02YTI1NGFiMDIwMGEifQ.ByaZmYhguwjFmzvkAXABwr4sFzMEPKID4clV7ygdpRNKbQ8FkoGmk0XWkyjW6q2pWOAeOQcwmw5p3uxkUtqMN8QsycW2Q4kFSlRxAu4SH69OqqypcaSmeBCI4UmewYjZykBEBpgBy0Ex9n9hkz9ty96Me-eRr4bVDS1CApEX7PVixLOMWkbgEySAA2S_JP9ZHdhzRqBfsLoIOK_DzZd-w8tGi29wPgjl4oTDmt_iokgg1lownQ0_UqhtvRLvGp_hJHMqiEpUryDhBLnHDWIPPofJf442ZeI1eQdAcN2vzdGAUxWpkztLfe1Q1_3WOBQvOs87pLGwprbcDSjbGnl1LmnPuj8tVRnBq4LtjMk1oC7v3gXDIjtyFNyPKNOFOrPDKJ-CIg25NwoGeZukQ4m2SrsSkAuGK2avuDZoU6Z9Krv9KvAw2JKJRfdvD2ICseIyaudgC2hRDk493G_auq2tM4nsqVWi_79owd2MhljqZazLGbBZAtS42soVTy4sUyy36mLbSTy5YXFVlXZysCLZEPa7ItNv3pGd4nQtww133GkOzgzBQzodW2EoWoxUnTssBk6L9c4f4wOiAOVGVH_1rAu-XloYHkvhlp--qO1nLXqjFUuS1Zgh8Vuof2L5LI1tKPR9T8SiqbvrNTlZAjnyOizK0R0c68JuJH1KulvD_t4
```

Comment:
Was this intended?

Reply:
Ou fuck... I revoked the key.... Thanks for noticing...